### ISSUE TITLE
Requirements for panda_moveit_config release

### ISSUE THREAD

**Issue by rhaschke** [frankaemika/franka_ros#193]
The new release of https://github.com/ros-planning/panda_moveit_config is blocked by a few things in `franka_ros`:
- [x] #199
- [x] #183
- [x] #196
- [x] ~~#190~~
  Covered by #199
- [x] ~~#186~~ -> e899240365797de6d53c222d6512b74c3a40268e
- [x] #194
- [x] ~~#173~~

I have merged these (and the first item from the nice-to-have list) into #188 for convenience.

### Nice to have
- [x] #191
- [x] Allow to use different hardware transmissions in Gazebo: https://github.com/frankaemika/franka_ros/commit/2514ae1990d51ca808e6c49d22d24ae7a99db532

### Optional
- [ ] #192

### Open issues
- [ ] GripperCommand doesn't signal convergence (but movements are correctly executed, see [#173](https://github.com/frankaemika/franka_ros/pull/173#issuecomment-933435669))
  - MoveIt controller manager fails with: `Controller is taking too long to execute trajectory`
  - `/franka_gripper/gripper_action/status` reports: 
    `Unexpected state transistion: The gripper not in HOLDING as expected`
- [x] #179
- [ ] #172

**Comment by rickstaa** [comment]
@rhaschke Thanks for collecting all our issues into an overview. One small suggestion can you change

> GripperCommand doesn't signal convergence (but movements are correctly executed) 

to 

> GripperCommand doesn't signal convergence (but movements are correctly executed) (see #173)

The problem is explained in detail in that pull request. Further, another related issue that should be mentioned is that #146, which was merged to the devel branch, broke the gripper actions (see #179).

**Comment by gollth** [comment]
For the record, @rhaschke shall we change PR #173 to Issue [#172](https://github.com/frankaemika/franka_ros/issues/172) on the checklist? (as discussed [here](https://github.com/frankaemika/franka_ros/pull/173#issuecomment-965023144))

**Comment by rickstaa** [comment]
>A few things block the new release of https://github.com/ros-planning/panda_moveit_configrankaemika/franka_ros/pull/191)
> * [ ]  Allow to use different hardware transmissions in Gazebo than the default effort transmission
>   
>   * [ ]  [Adds the 'transmission' arg to the 'panda_arm.urdf.xacro' file #157](https://github.com/frankaemika/franka_ros/pull/157)
>   * [ ]  [Adds position and velocity motion generators to FrankaHWSim #181](https://github.com/frankaemika/franka_ros/pull/181)

@rhaschke Just for tracking purposes. The motion generators have been implemented in https://github.com/frankaemika/franka_ros/commit/2514ae1990d51ca808e6c49d22d24ae7a99db532.
